src/cloud/ops.rs

```diff
@@ -141,43 +148,43 @@ pub async fn create_cloud_instance(
     client: &CloudClient,
     instance: &CloudInstanceCreate,
 ) -> anyhow::Result<()> {
-    let cred_path = credentials::path(&instance.name)?;
-    if cred_path.exists() {
-        anyhow::bail!("File {} exists; abort.", cred_path.display());
-    }
+    let url = format!("orgs/{}/instances", instance.org_slug);
     let instance: CloudInstance = client
-        .post("instances/", serde_json::to_value(instance)?)
+        .post(url, serde_json::to_value(instance)?)
         .await?;
-    let instance = wait_instance_create(instance, client, false).await?;
-    write_credentials(&cred_path, instance).await?;
```

Comment:
I think removing this line is a regression, no? I see one for linking the cloud instance, but nothing for creating it.

Reply:
The other one should be removed too. Thanks for catching that.